.gitmodules

```diff
@@ -43,3 +43,6 @@
 [submodule "libcouchbase"]
 	path = plugins/couchbase/libcouchbase
 	url = https://github.com/hpcc-systems/libcouchbase.git
+[submodule "plugins/redis/hiredis"]
+	path = plugins/redis/hiredis
+	url = https://github.com/redis/hiredis.git
```

Comment:
As with the other submodules, we need to clone this to ensure that it continues to exist